samples/ResourceGroup/require-resource-location-match-resourceGroup/README.md

```diff
@@ -0,0 +1,24 @@
+# Require Resource Location to match Resource Group
+
+Denies deployment of resource to Resource Group is location does not match that of the Resource Group. This is useful to prevent accidental misconfiguration leading to increased cost and latency.
+
+## Try in the Azure Portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FResourceGroup%2Frequire-resource-location-match-resourceGroup%2Fazurepolicy.json)
+
+## Try with PowerShell
+
+````powershell
+$definition = New-AzPolicyDefinition -Name "require-resource-location-match-resourceGroup" -DisplayName "Audit Resource Locks on Resource Groups based on Tags" -description "Audits all Resource Groups that have a specific Tag, for the CanNotDelete Resource Lock." -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/ResourceGroup/require-resource-location-match-resourceGroup/azurepolicy.rules.json' -Mode Indexed
```

Comment:
>"Audit Resource Locks on Resource Groups based on Tags [](start = 104, length = 54)

please update the display name and description for the powershell example

Reply:
done. Thanks for the catch.